app/routes/store.tsx

```diff
@@ -118,7 +118,7 @@ export const useReactStoreDocsConfig = () => {
   const params = useParams()
   const version = params.version!
   const framework =
-    params.framework || localStorage.getItem('framework') || 'react'
+    params.framework || localStorage.getItem(storeLocalStorageKey) || 'react'
```

Comment:
Just checking on this again - could we keep it as `framework` rather than `framework-store` and `framework-form`?